packs/kingmaker-bestiary/oleg.json

```diff
@@ -270,11 +265,10 @@
                 "containerId": null,
                 "damage": null,
                 "description": {
-                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> envision;</p>\n<p><strong>Requirement</strong> You are trained in Medicine.</p>\n<hr />\n<p>This lock of hair is wrapped around a twig twisted into the shape of a heart. Popular among many of Restov's explorers, traders, and mercenaries, often a person carries several lover's knots on their person in case of emergency. While the original lover's knots were specific rewards granted by a particularly flirtatious Swordlord to her favored agents, time has seen these talismans become much more widespread through southern Brevoy. When you activate a lover's knot, you regain [[/r 2d6[healing]]]{2d6 Hit Points}. If you have the @UUID[Compendium.pf2e.feats-srd.Item.Battle Medicine] feat, you instead regain [[/r (2d6+7)[healing]]]{2d6+7 Hit Points}.</p>"
+                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> envision;</p>\n<p><strong>Requirement</strong> You are trained in Medicine.</p><hr /><p>This lock of hair is wrapped around a twig twisted into the shape of a heart. Popular among many of Restov's explorers, traders, and mercenaries, often a person carries several lover's knots on their person in case of emergency. While the original lover's knots were specific rewards granted by a particularly flirtatious Swordlord to her favored agents, time has seen these talismans become much more widespread through southern Brevoy. When you activate a lover's knot, you regain @Damage[2d6[healing]]{2d6 Hit Points}. If you have the @UUID[Compendium.pf2e.feats-srd.Item.Battle Medicine] feat, you instead regain @Damage[(2d6+7)[healing]]{2d6+7 Hit Points}.</p>"
```

Comment:
Should the source item be updated too?

Reply:
I only changed the item on the actor to use the updated version that was in the system.